rasgoql/rasgoql/primitives/enums.py

```diff
@@ -14,26 +14,6 @@
 def _wrap_in_quotes(string: str) -> str:
     return f"'{string}'"
 
-class DWType(Enum):
-    """
-    Supported Data Warehouses
-    """
-    BIGQUERY = 'BIGQUERY'
-    #FUTURE: POSTGRES = 'POSTGRES'
-    SNOWFLAKE = 'SNOWFLAKE'
-
-def check_data_warehouse(input_value: str):
-    """
-    Warn if an incorrect Data Warehouse is passed
-    """
-    supported_dws = _wrap_in_quotes("', '".join([e.value for e in DWType]))
-    try:
-        output_value = DWType[input_value.upper()].value
-    except Exception:
-        raise ParameterException(f'data_warehouse parameter accepts values: {supported_dws}')
-    return output_value
-
-
```

Comment:
This function is not used, would not do random cleanup, but because the `DWType` would need expanded / tested somehow it is simpler this way.